libs/platform/src/lib/components/link/link.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { LinkComponent } from './link.component';
+import { FundamentalNgxCoreModule } from '@fundamental-ngx/core';
+
+@NgModule({
+    declarations: [LinkComponent],
+    imports: [CommonModule, FundamentalNgxCoreModule],
+    exports: [LinkComponent]
```

Comment:
Can you double check this line? where you imports whole ngxCore ? you should be fine just importing the link module now?

Reply:
resolved this. now Importing only LinkModule from FundamentalNgxCoreModule,